hudi-common/src/test/java/org/apache/hudi/common/testutils/reader/HoodieFileGroupReaderTestUtils.java

```diff
@@ -119,6 +119,8 @@ public HoodieFileGroupReader<IndexedRecord> build(
           fileSlice,
           schema,
           schema,
+          Option.empty(),
+          null,
```

Comment:
Instead of null, can we change the argument to use `Option`? Or pass the metaclient from the test.